lib/services/role.go

```diff
@@ -810,25 +811,41 @@ func (result *EnumerationResult) filtered(value bool) []string {
 	return filtered
 }
 
-// Denied returns all explicitly denied users.
+// Check if the given entity is allowed.
```

Comment:
```suggestion
// IsAllowed checks if the given entity is allowed.
```

nit: it's customary for godocs to begin with the name of the identifier they are documenting.